test/transform-tests.js

```diff
@@ -15,6 +19,8 @@ describe("compiler.transform", () => {
       const code = files[absPath];
       const ast = options.parse(code);
 
+      options.moduleId = makeUniqueId("module", code);
```

Comment:
It's a little unfortunate that `transform` can't come up with a unique `module` name on its own, but that's ok because only the Babel plugin currently uses `transform`, and the Babel plugin sets `options.moduleId` explicitly.